README.md

```diff
@@ -30,21 +30,18 @@
 
 # OpenVINO Backend
 
-**Note: OpenVINO backend is beta quality. As a result you may
-encounter performance and functional issues that will be resolved in
-future releases.**
+**Note: OpenVINO backend is now considered of general availability. It is co-maintained by Nvidia and Intel.**
```

Comment:
```suggestion
**Note: The OpenVINO backend has been promoted out of Beta and is now ready for production use. It is co-maintained by Nvidia and Intel.**
```

Reply:
I don't think we need this note at all. Simple just state in the description below that this repo is co-maintained by Intel and NVIDIA and remove all mentions of beta or GA. We can make an official announcement in the release notes.

Reply:
will make the change part of release notes / announcments.